go-controller/pkg/ovn/master_test.go

```diff
@@ -1519,6 +1520,7 @@ func newNodeSNAT(uuid, logicalIP, externalIP string) *nbdb.NAT {
 }
 
 func TestController_syncNodes(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
```

Comment:
is this necessary?

Reply:
yes, the tests here are not inside the ginkgo assertion, so we have to register a fail handler